telegram/audio.py

```diff
@@ -46,9 +46,9 @@ class Audio(TelegramObject):
     def __init__(self,
                  file_id,
                  duration,
-                 performer='',
-                 title='',
-                 mime_type='',
+                 performer=None,
+                 title=None,
+                 mime_type=None,
                  file_size=0,
```

Comment:
We have that in a few places... I don't really know...
It's the same on with the booleans on Message.